### DISCUSSION TITLE
Next release date?

### DISCUSSION THREAD

**Discussion by void-rooster** [mie-lab/trackintel#486] [General]
I'm on a project that uses trackintel as part of a preprocessing task. We'd like to create a virtual environment with a pinned version of the library that has #474 . When will the next version be released?

**Comment by hongyeehh** [comment]
Hey, 
Sorry for the delay; we did not see the message in Discussions. We just released the newest version of trackintel; you can now access it from pip or conda. Thanks for using trackintel!
